app/stacks/uk-south/appeals-service/secrets.tf

```diff
@@ -34,3 +34,15 @@ resource "azurerm_key_vault_secret" "appeals_mongo_db_connection_secret" {
 
   tags = local.tags
 }
+
+
+resource "azurerm_key_vault_secret" "appeals_horizon_service_bus_key" {
+  count           = var.is_dr_deployment ? 1 : 0
+  name            = "appeals-horizon-pub-password"
+  value           = module.app_services[0].service_bus_horizon_publish_key
```

Comment:
not sure on this one, the service bus is defined in components so am referencing from it's outputs here using module.app_services but that appears to be an array, which feels off, is this correct?